lib/reporters/html/template-default.hbs

```diff
@@ -28,7 +28,7 @@
     <div class="row">
       {{#with summary}}
         <div class="col-md-3">Collection</div><div class="col-md-9">{{collection.name}}</div>
-        {{#if collection.description}}<div class="col-md-3">Description</div><div class="col-md-9">{{collection.description}}</div>{{/if}}
+        {{#if collection.description}}<div class="col-md-3">Description</div><div class="col-md-9">{{collection.description}}&nbsp;</div>{{/if}}
```

Comment:
For empty collection descriptions, the row alignment is horribly skewed, such that the next line begins exactly where the description should have been.